recipes/newrelic/infrastructure/agent-control/debian.yml

```diff
@@ -332,20 +332,7 @@ install:
             AGENT_SET="infra-agent"
           fi
           # Proxy configuration
-          if [ ! -z "{{.NEW_RELIC_AGENT_CONTROL_PROXY_URL}}" ] ; then
-            AC_PROXY_ARGS="--proxy-url {{.NEW_RELIC_AGENT_CONTROL_PROXY_URL}}"
-            if [ ! -z "{{.NEW_RELIC_AGENT_CONTROL_PROXY_CA_BUNDLE_FILE}}" ] ; then
-              AC_PROXY_ARGS="$AC_PROXY_ARGS --proxy-ca-bundle-file {{.NEW_RELIC_AGENT_CONTROL_PROXY_CA_BUNDLE_FILE}}"
-            fi
-            if [ ! -z "{{.NEW_RELIC_AGENT_CONTROL_PROXY_CA_BUNDLE_DIR}}" ] ; then
-              AC_PROXY_ARGS="$AC_PROXY_ARGS --proxy-ca-bundle-dir {{.NEW_RELIC_AGENT_CONTROL_PROXY_CA_BUNDLE_DIR}}"
-            fi
-            if [ "{{.NEW_RELIC_AGENT_CONTROL_PROXY_IGNORE_SYSTEM}}" == "true" ] ; then
-              AC_PROXY_ARGS="$AC_PROXY_ARGS --ignore-system-proxy"
-            fi
-          else
-            AC_PROXY_ARGS=""
-          fi
+          AC_PROXY_ARGS="--proxy-url={{.NEW_RELIC_AGENT_CONTROL_PROXY_URL}} --proxy-ca-bundle-file={{.NEW_RELIC_AGENT_CONTROL_PROXY_CA_BUNDLE_FILE}} --proxy-ca-bundle-dir={{.NEW_RELIC_AGENT_CONTROL_PROXY_CA_BUNDLE_DIR}} --ignore-system-proxy={{.NEW_RELIC_AGENT_CONTROL_PROXY_IGNORE_SYSTEM}}"
```

Comment:
would make sense to clarify that "if no value set the flag is going to be ignored"?

Reply:
Sure! I'm adding a comment to clarify!

Reply:
Updated here: [c6d7267](https://github.com/newrelic/open-install-library/pull/1287/commits/c6d7267ce13492f5f2da5407f434e1fb8e78af8c)